### PR TITLE
add batch of string ops

### PR DESCRIPTION
**Pull request by eellison**
First batch of https://github.com/pytorch/pytorch/issues/20769, handles `isupper`, `islower`, `isdigit`, `isspace`, `isalnum`, `isalpha`, `upper`, `lower`

### CODE REVIEW

File: test/test_jit.py [hunk 1]
```diff
@@ -12074,14 +12074,29 @@ def fn(x):
...
+        def test_str_is(s):
+            # type: (str) -> Tuple[bool, bool, bool, bool, bool, bool]
+            return s.isupper(), s.islower(), s.isdigit(), s.isspace(), \
+                s.isalnum(), s.isalpha()
+
+        def test_str_to(s):
+            # type: (str) -> Tuple[str, str]
+            return s.upper(), s.lower()
+
+        inputs = ["", "12a", "!B", "12", "a", "B", "aB", "$12", "B12", "AB ",
```
Comment (driazati) [inline test/test_jit.py]: needs a >1 char string of lowers: `"aaaa"` or something

**Review by driazati — APPROVED** [review]

**Comment by eellison** [comment]
test failures unrelated

**Comment by ezyang** [comment]
This broke Windows. 

```
00:57:52          C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\torch\csrc\jit\register_prim_ops.cpp(1939): error C2039: 'isalpha': is not a member of 'std' [C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\caffe2\caffe2.vcxproj]
00:57:52          C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\torch\csrc\jit\register_prim_ops.cpp(1940): error C2039: 'isalpha': is not a member of 'std' [C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\caffe2\caffe2.vcxproj]
00:57:52          C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\torch\csrc\jit\register_prim_ops.cpp(1940): error C2039: 'isupper': is not a member of 'std' [C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\caffe2\caffe2.vcxproj]
```

**Comment by eellison** [comment]
yep fix in here https://github.com/pytorch/pytorch/pull/20842

**Pull request closed without merging**